<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualizer3Dvtk</class>
 <widget class="QMainWindow" name="Visualizer3Dvtk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Visualizer (VTK)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/logo.png</normaloff>Icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="timeStep_Slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevStep_Button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Icons/icon-ios7-arrow-left-128.png</normaloff>Icons/icon-ios7-arrow-left-128.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextStep_Button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Icons/icon-ios7-arrow-right-128.png</normaloff>Icons/icon-ios7-arrow-right-128.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeStep_LineEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="plot_Widget" native="true">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>500</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="plotWidget_layout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Available Data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTabWidget" name="tabWidget">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab">
                 <attribute name="title">
                  <string>3D Fields</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="leftMargin">
                   <number>7</number>
                  </property>
                  <property name="topMargin">
                   <number>7</number>
                  </property>
                  <property name="rightMargin">
                   <number>7</number>
                  </property>
                  <property name="bottomMargin">
                   <number>7</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Select fields:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QListView" name="availableFields_listView">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="addToRender_Button">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Add To Render</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_2">
                 <attribute name="title">
                  <string>Domain Fields</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="leftMargin">
                   <number>7</number>
                  </property>
                  <property name="topMargin">
                   <number>7</number>
                  </property>
                  <property name="rightMargin">
                   <number>7</number>
                  </property>
                  <property name="bottomMargin">
                   <number>7</number>
                  </property>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Visualization Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTabWidget" name="tabWidget_2">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab_3">
                 <attribute name="title">
                  <string>List of fields</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <property name="leftMargin">
                   <number>7</number>
                  </property>
                  <property name="topMargin">
                   <number>7</number>
                  </property>
                  <property name="rightMargin">
                   <number>7</number>
                  </property>
                  <property name="bottomMargin">
                   <number>7</number>
                  </property>
                  <item>
                   <widget class="QListWidget" name="fieldsToRender_listWidget"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="render_pushButton">
                    <property name="text">
                     <string>Render</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="screenshotButton">
                    <property name="text">
                     <string>Screenshot</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_4">
                 <attribute name="title">
                  <string>Settings</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="leftMargin">
                   <number>7</number>
                  </property>
                  <property name="topMargin">
                   <number>7</number>
                  </property>
                  <property name="rightMargin">
                   <number>7</number>
                  </property>
                  <property name="bottomMargin">
                   <number>7</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Background color:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QRadioButton" name="black_bg_radioButton">
                      <property name="text">
                       <string>Black</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="white_bg_radioButton">
                      <property name="text">
                       <string>White</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionMake_video">
   <property name="text">
    <string>Create animation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionParticle_Tracker">
   <property name="text">
    <string>Particle Tracker</string>
   </property>
  </action>
  <action name="actionSimulation_parameters">
   <property name="text">
    <string>Simulation parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
